fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/DefaultOCFLObjectSessionFactory.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.persistence.ocfl.impl;
+
+import edu.wisc.library.ocfl.api.MutableOcflRepository;
+import edu.wisc.library.ocfl.core.OcflRepositoryBuilder;
+import edu.wisc.library.ocfl.core.mapping.ObjectIdPathMapperBuilder;
+import edu.wisc.library.ocfl.core.storage.FileSystemOcflStorage;
+import org.fcrepo.persistence.ocfl.api.OCFLObjectSession;
+import org.fcrepo.persistence.ocfl.api.OCFLObjectSessionFactory;
+
+import java.io.File;
+
+import static java.lang.System.getProperty;
+
+/**
+ * A default implemenntation of the {@link org.fcrepo.persistence.ocfl.api.OCFLObjectSessionFactory} interface.
+ *
+ * @author dbernstein
+ * @since 6.0.0
+ */
+public class DefaultOCFLObjectSessionFactory implements OCFLObjectSessionFactory {
+
+    private static final File DEFAULT_STAGING_DIR = new File(getProperty("fcrepo.ocfl.staging.dir",
+            getProperty("java.io.tmpdir") + File.separator + "fcrepo-ocfl-staging"));
```

Comment:
There is a constant available in apache commons for the default tmpdir property if you'd like to use it http://commons.apache.org/proper/commons-lang/javadocs/api-release/org/apache/commons/lang3/SystemUtils.html#JAVA_IO_TMPDIR

I'm not sure if it would cause any problems if the tmpdir ended with a `/`. It might be a little safer to use `Paths.get(getProperty("java.io.tmpdir"), "fcrepo-ocfl-staging")`

Reply:
+1